README.md

```diff
@@ -23,6 +23,11 @@ an example file:
 
 ```
 DEBUG = True
+
+VERSIONS = {
+    'css': 0,
+    'js': 0,
+}
```

Comment:
I'd prefer to calculate this dynamically (e.g. from the hash of the file), instead of having to remember to update the version.

Reply:
This didn't turn out to be too hard; webpack comes with the ability to add the file's hash to its file name, and there was an already-existing webpack plugin for generating a json manifest of the hashes. Deploying!